ui/src/app/pages/analysis/runtime-panel.spec.tsx

```diff
@@ -159,6 +159,55 @@ describe('RuntimePanel', () => {
     expect(runtimeApiStub.runtime.gceConfig.machineType).toEqual('n1-standard-4');
   });
 
+  it('should create runtime with preset values instead of getRuntime values if configurationType is GeneralAnalysis', async() => {
+    const gceConfig = defaultGceConfig();
```

Comment:
opt: imo maybe slightly more readable as:

```
{
  ...defaultGceConfig(),
  machineType: ...,
  diskSize: ...
```

Also, if you take this suggestion, you may not need the `gceConfig` variable either, can inline the whole thing below